wagtail/admin/checks.py

```diff
@@ -188,8 +188,17 @@ def wagtail_admin_base_url_check(app_configs, **kwargs):
 @register("file_overwrite")
 def file_overwrite_check(app_configs, **kwargs):
     from django.conf import settings
-
-    file_storage = getattr(settings, "DEFAULT_FILE_STORAGE", None)
+    from django import VERSION as DJANGO_VERSION
+
+    if DJANGO_VERSION >= (4, 2):
```

Comment:
@gasman Something like this then?

```suggestion
    if (4, 2) <= DJANGO_VERSION < (5, 1):
```

Reply:
That would work, but I think it might be more readable to reorder the cases so that `DJANGO_VERSION >= (5, 1)` comes first, then `elif DJANGO_VERSION >= (4, 2)`.

Reply:
👍 Done.

The commit history is very messy now. What would you prefer in this case? A new PR? Squashing/renaming commits?

Reply:
That's all fine, thanks! I'll deal with squashing the commits when I merge (which I'll do now - everything looks good).